packages/cmf/src/middlewares/http/middleware.js

```diff
@@ -101,10 +101,16 @@ export function getMethod(action) {
 
 export function mergeConfiguredHeader(config) {
 	// still need to keep the previous header added by action
-	return options => ({
-		...options,
-		headers: { ...options.headers, ...DEFAULT_HTTP_HEADERS, ...config.headers },
-	});
+	return options => {
+		const headerMergedConfig = {
+			...options,
+			headers: { ...DEFAULT_HTTP_HEADERS, ...options.headers, ...config.headers },
```

Comment:
Shouldn't be more like 
`{ ...DEFAULT_HTTP_HEADERS, ....config.headers, ...options.headers }`
to allow specific options to override global config ?

Reply:
you're right about that options are the one coming from the action so it should be the last one overriding